.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image: ubuntu:latest
+
+tasks:
+  - before: echo "..."
```

Comment:
What is this about?

Reply:
just some debugging. I wasn't sure when this runs, still aint